themes/medius/scss/variables.scss

```
@@ -171,6 +166,27 @@ $white-35: rgba(255, 255, 255, .35);
 $white-20: rgba(255, 255, 255, .2);
 $white-10: rgba(255, 255, 255, .1);
 
+// Yellows are deprecated. Use Mustard instead.
+$yellow-10: #FDF8ED; 
+$yellow-20: #FCE9CF; 
+$yellow-30: #ffc978; 
+$yellow-40: #E9A449; 
+$yellow-60: #935E25; 
+$yellow-70: #663F19; 
+$yellow-80: #38210A; 
+
+// Colors by category
+$business: $dark-mustard;
+$conferences: $dark-turquoise;
+$design: $dark-coral;
+$development: $dark-purple;
+
+$business-bg: $mustard;
+$conferences-bg: $turquoise;
+$design-bg: $coral;
+$development-bg: $purple;
```

```suggestion
$business: $dark-mustard;
$conferences: $dark-turquoise;
$design: $dark-coral;
$development: $dark-purple;

$business-bg: $mustard-40;
$conferences-bg: $turquoise;
$design-bg: $coral-40;
$development-bg: $purple-60;
```

You should use the right name here